### PR TITLE
Update documentation to give folks more info on what we can’t accept

### PR DESCRIPTION
**Pull request by taquitos**
We have some small limitations with how the project is structured. This PR updates the documentation and explains it better.

### Checklist
- [x] All new and existing tests pass
- [x] I've read the [Contribution Guidelines](https://github.com/google/science-journal-ios/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGE_LIMITATIONS.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# What changes we can't accept
+
+We wish we could accept all changes, especially changes that make the organization or readability of the project better. Unfortunately we can't, due to technical limitations 😢.
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: "While we wish we could accept all changes (especially changes that make the organization or readability of the project better), we can't due to technical limitations."

File: CHANGE_LIMITATIONS.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## What is up with this weird module map, or strange import statement 🤷‍♂️?
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: Remove the comma after module map

File: CHANGE_LIMITATIONS.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+We host our source code inside Google and treat that as the “source of truth”. The internal build toolchain and dependency manager are not cocoapods. In order to be able to work with both internal and external tooling, we’ve had to make some concessions with how the project is set up. There are a number of considerations you must make if you’re thinking about changing things that look non-standard.
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: capitalize CocoaPods

Comment (garrettmurray) [reply CHANGE_LIMITATIONS.md]: Change "how the project is set up" > "how the project is configured"

File: CHANGE_LIMITATIONS.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+### Module maps and import statements
+
+We use custom module maps that link directly to various dependencies like [SSOAuth](https://github.com/google/science-journal-ios/blob/master/ModuleMaps/SSOAuth.modulemaps/module.modulemap). We can’t modify these as our internal dependency management tool has some requirements that necessitate the current import format. You can see that come into play where imports are prefixed with things like “[googlemac_iPhone](https://github.com/google/science-journal-ios/blob/master/ScienceJournal/Accounts/AccountsManager.swift#L19)”. Sadly, we must leave these in.
+
+### Updating Swift
+
+We want to be running the latest Swift 🏃‍♀️ as much as anybody. At this time we can’t accept Swift version update PRs. Our internal source of truth must be updated first. Because the internal source of truth has more code than the open source project, we’ll have to do the migration, and test it internally to make sure all of our tooling works- inside and outside of Google.
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: works—inside and outside of Google

(as in make that an emdash that connects works and inside)

File: CHANGE_LIMITATIONS.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### Podfile updates
+
+Adding new pods isn’t out of the question, but please be mindful as we only support Apache 2.0, and MIT licensed software.
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: remove comma after 2.0

File: CHANGE_LIMITATIONS.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+Updating the pods that are used is a different story too, that is, it's unlikely we can accept any pod updates. We have more internal requirements that must be satisfied so we limit podfile updates to new features, bug fixes, and security updates that impact us.
```
Comment (garrettmurray) [inline CHANGE_LIMITATIONS.md]: "It is unlikely we'd be able to approve the removal of existing pods. We have more internal requirements that much be satisfied so we limit podfile updates to new features, bug fixes and security updates that impact us."

File: CHANGE_LIMITATIONS.md (hunk 1 above)
Comment (taquitos) [reply CHANGE_LIMITATIONS.md]: 💯 I like this wording way better

File: CHANGE_LIMITATIONS.md (hunk 3 above)
Comment (taquitos) [reply CHANGE_LIMITATIONS.md]: 👍

File: CHANGE_LIMITATIONS.md (hunk 6 above)
Comment (taquitos) [reply CHANGE_LIMITATIONS.md]: Gonna use very similar language, but adding in dat oxford comma `bug fixes*,* and security`

File: CHANGE_LIMITATIONS.md (hunk 4 above)
Comment (taquitos) [reply CHANGE_LIMITATIONS.md]: 🐐

File: CHANGE_LIMITATIONS.md (hunk 5 above)
Comment (taquitos) [reply CHANGE_LIMITATIONS.md]: 👍

**Pull request merged**